app/src/main/java/max/music_cyclon/service/LibraryService.java

```diff
@@ -168,14 +175,14 @@ protected void onHandleIntent(Intent intent) {
             e.printStackTrace();
         }
 
-        updater.showMessage("Musik aktualisiert");
+        updater.showMessage(getResources().getString(R.string.music_updated));
 
+        // I don't want to support proprietary things
```

Comment:
Hahah :+1:

Reply:
Ops sorry XD I forgot to delete it before committing

Reply:
Its ok though :) I'm no longer the only user and I dont user poweramp anymore because I'm google apps free :)

Reply:
Added a comment on it